.github/workflows/clair-in-ci-db.yaml

```diff
@@ -0,0 +1,59 @@
+name: Clair in CI DB
+
+# This workflow builds latest version of clair-in-ci-db and
+# pushes it to app-studio registry
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: clair-in-ci-db
```

Comment:
Name should correspond to the one we are using in build definitions:
https://github.com/redhat-appstudio/build-definitions/blob/main/tasks/get-clair-scan.yaml#L14
`clair-in-ci-db` > `clair-in-ci`